ArdusatSDK.cpp

```diff
@@ -1,28 +1,40 @@
 /**
  * @file   ArdusatSDK.cpp
- * @Author Ben Peters (ben@ardusat.com)
+ * @author Ben Peters (ben@ardusat.com)
+ * @author Sam Olds (sam@ardusat.com)
  * @date   December 3, 2014
  * @brief  Implements ArdusatSDK generic sensor reading and configuration for Space Kit Sensors.
+ *
+ * Provides a unifying wrapper of sensor specific functionality to provide a
+ * consistent interface to interact with each type of sensor.
  */
 
 #include <stdio.h>
 #include <string.h>
 #include "ArdusatSDK.h"
 
-bool ARDUSAT_SHIELD = false;
+bool MANUAL_CONFIG = false;
```

Comment:
disregard the comment re: brief comment here - saw you did that in the header file. But definitely add a description in the README